test/modal-manager.test.ts

```diff
@@ -322,16 +328,14 @@ describe('Modal Manager', () => {
     const closeButton = modal?.shadowRoot?.querySelector(
       '.close-button'
     ) as HTMLElement;
-    const activeElement = modal?.shadowRoot?.activeElement as HTMLElement;
-
-    expect(activeElement).to.equal(closeButton);
+    expect(modal?.shadowRoot?.activeElement).to.equal(closeButton);
 
     // Tab again
     el.dispatchEvent(tabEvent);
     await elementUpdated(el);
 
     // Should be only one tabbable element
-    expect(activeElement).to.equal(closeButton);
+    expect(modal?.shadowRoot?.activeElement).to.equal(closeButton);
```

Comment:
These existing tests were incorrectly checking the same element after each tab, rather than checking the live value of the active element. Just fixing them in passing.